server/src/main/resources/application.example.yml

```diff
@@ -67,9 +67,16 @@ seata:
       server-addr: http://localhost:2379
       key: seata.properties
   registry:
-    # support: nacos 、 eureka 、 redis 、 zk  、 consul 、 etcd3 、 sofa
+    # support: nacos 、 eureka 、 redis 、 zk  、 consul 、 etcd3 、 sofa 、 namingserver
     type: file
     preferred-networks: 30.240.*
+    metadata:
+      weight: 1
+    namingserver:
+      server-addr: 127.0.0.1:8080,127.0.0.1:8081
```

Comment:
why port is 8080?It is possible to write only one address here, because it is only an example.